packages/scripts/tests/fixtures/projects/01-valid/microapp.json

```diff
@@ -1,4 +1,3 @@
 {
-  "name": "01-valid",
-  "entryComponent": "./index.js"
+  "name": "01-valid"
```

Comment:
We no longer need this fixtures folder as we don't need to modify the next.config file anymore.